recipes/gocryptfs/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "gocryptfs" %}
+{% set version = "2.6.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/rfjakob/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: f972d95d8e295e177cb0ab20598d7a52d77c1b982c208bfc9417bde8726d80ae
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ stdlib('c') }}
+    - {{ compiler('go') }}
```

It seems like this doesn't use a C compiler
```suggestion
    - {{ compiler('go-nocgo') }}
```